README.md

```diff
@@ -182,32 +182,41 @@ In order to start them, some packages must be installed and started. Here a quic
 
 ```shell
 # Install ARA with server dependencies
-pip install --user "ara[server]"
+pip install "ara[server]"
 # Setup ARA database
 ara-manage migrate
 # Start ARA server. This process can be started in background or in other shell terminal
 ara-manage runserver ip:port
 ```
 
+If the requests to ARA server fail with a message like the next one, it means that the server address must be allowed:
+
+```
+2021-09-02 07:13:48,715 ERROR django.security.DisallowedHost: Invalid HTTP_HOST header: '10.74.1.5:8000'. You may need to add '10.74.1.5' to ALLOWED_HOSTS.
+2021-09-02 07:13:48,732 WARNING django.request: Bad Request: /api/
+```
+
+To fix it run:
+
+```
+export ARA_ALLOWED_HOSTS="['10.74.1.5']"
+```
```

Comment:
this would block localhost connection to the ara server tho.
Maybe we could just be lazy with `ARA_ALLOWED_HOSTS=['*']` ?

Reply:
Ok, I will add a new line with this option for lazy admins XD